src/Processors/Transforms/Streaming/AggregatingHelper.h

```diff
@@ -38,16 +38,13 @@ Chunk mergeAndSpliceAndConvertBucketsToChunk(
     ManyAggregatedDataVariants & data, const AggregatingTransformParams & params, const std::vector<Int64> & buckets);
 
 /// Only used for emit changelog
-/// @brief Based on new/updated groups @p retracted_data , only convert the state of changed groups (retracted: last state, aggregated: current state)
-///  \data: current aggregated state of all groups
-///  \retracted_data: only have last state of changed groups (i.e. new/updated/deleted)
+/// @brief only convert the state of changed groups (retracted: last state, aggregated: current state)
+///  \data: current aggregated state of all groups (contains retracted states and updated states)
```

Comment:
```
/// Changelog chunk converters are used for changelog emit. They can return a pair of chunks : one 
/// for retraction and one for updates. And those 2 chunks are expected to be passed to downstream
/// consecutively otherwise the down stream aggregation result may not be correct or emit intermediate 
/// incorrect result
/// 
```